app/decorators/book_decorator.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class BookDecorator < Draper::Decorator
+  SHORT_DESCRIPTION_LENGTH = 250
+
+  delegate_all
+
+  def all_authors
+    authors.map(&:name).join(',')
```

Comment:
map is bad usage with relations, use pluck

Reply:
fixed